dotnet/src/SemanticKernel/TemplateEngine/PromptTemplateEngine.cs

```diff
@@ -85,8 +85,10 @@ public async Task<string> RenderAsync(IList<Block> blocks, SKContext context)
             }
         }
 
-        // TODO: remove PII, allow tracing prompts differently
-        this._log.LogDebug("Rendered prompt: {0}", result);
+        // TODO: allow tracing prompts differently
```

Comment:
Can you remove the TODO here? I think your LogTrace strategy handles this. I think it's sufficient to have a consistent comment everywhere like below ("// Sensitive data, logging as trace, disabled by default")